cmd/zoekt-mirror-github/main.go

```diff
@@ -309,9 +316,10 @@ func cloneRepos(destDir string, repos []*github.Repository) error {
 			"zoekt.github-subscribers": itoa(r.SubscribersCount),
 			"zoekt.github-forks":       itoa(r.ForksCount),
 
-			"zoekt.archived": marshalBool(r.Archived != nil && *r.Archived),
-			"zoekt.fork":     marshalBool(r.Fork != nil && *r.Fork),
-			"zoekt.public":   marshalBool(r.Private == nil || !*r.Private),
+			"zoekt.archived":   marshalBool(r.Archived != nil && *r.Archived),
+			"zoekt.fork":       marshalBool(r.Fork != nil && *r.Fork),
+			"zoekt.public":     marshalBool(r.Private == nil || !*r.Private),
+			"zoekt.visibility": r.GetVisibility(),
```

Comment:
can you remove this change. We don't consume this metadata so we don't need to add this.